docs/site/Relation-generator.md

```diff
@@ -0,0 +1,110 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
+---
+
+### Prerequisites
+
+Important: Before running this generator, you must create an application using
+the application generator. Then you must run the command from the root directory
+of the application. The models involved in the relation must exist before
+running this generator. The datasource involved in the relation must exist
+before running this generator. The repositories involved in the relation must
+exist before running this generator.
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+Adds a new `Relation` between existing source and target models in a LoopBack
+application.
+
+```sh
+lb4 relation [options]
+```
+
+### Options
+
+- `--help`: Print the generator’s options and usage.
+- `--skip-cache`: Do not remember prompt answers. Default: `false`.
+- `--skip-install`: Do not automatically install dependencies. Default: `false`.
+- `--force-install`: Fail on install dependencies error. Default: `false`.
+- `--relationType`: Relation type.
+- `--sourceModel`: Source model.
+- `--destinationModel`: Destination model.
+- `--foreignKeyName`: Destination model foreign key name.
+- `--relationName`: Relation name.
+
+### Arguments
+
+Defining lb4 relation in one command line interface (cli):
+
+```sh
+\$ lb4 relation --sourceModel=<sourceModel>
```

Comment:
Nitpick: can you remove the `\`?
Personally, I'd remove this whole block of code so it's more in line with the other generators' docs, but it's up to you.

Reply:
Removed \$ from the doc file.